source/manuals/programming-languages/nodejs/index.html.md.erb

```diff
@@ -4,43 +4,75 @@ title: Using Node.js at GDS
+GDS Way Forum. Topics like this are discussed at our monthly meetings. To contact
+the forum [email](mailto:programme-technical-leaders@digital.cabinet-office.gov.uk) explaining
+the need and linking to whatever else might be relevant. The [programming language recommendations](../../../standards/programming-languages.html#javascript)
+section has more detail on how we currently use Node.js at GDS and why this
 decision has been made.
 
-## Guidance
-The guidance set out here should be followed in conjunction with the advice on the main [programming languages manual page](../../programming-languages.html).
 
-If you want to add guidance on Node.js please see [Updating this manual](#updating-this-manual) below.
+## Introduction
 
-### Style
-#### Formatting and linting
-To maintain a consistent and readable JavaScript codebase we use [StandardJS]. This includes a built in formatter, linter and automatic code fixer (auto-fix). To make sure everything stays clean we recommend running this tool before each commit. It is possible to setup a pre-commit hook to automate this process. When [StandardJS] automatically fixes errors it has clear and concise messaging and it also tells you when an automatic fix isn't possible.
+The guidance set out here should be followed in conjunction with the advice on
+the main [programming languages manual page](../../programming-languages.html).
 
-We’ve encapsulated our recommended process into a separate [repository](https://github.com/alphagov/nodejs-starter) where everything is controlled by a few lines of config within your `package.json` file. Once you have run `npm install`, any time you make a commit, [StandardJS] runs. It will then format and lint your code automatically. [StandardJS] will auto-fix obvious errors then continue with your commit. If there’s something it can’t fix, then the commit fails with a useful error message such as:
+The advice here is specifically about Node.js. Generic guidelines on writing
+code are out of scope.
 
-```
-❯ Running tasks for *.js
-   ✖ standard --fix
-     → standard: Use JavaScript Standard Style (https://standardjs.com)
-     git add
-✖ standard --fix found some errors. Please fix them and try committing again.
-  ~/nodejs-starter/index.js:5:9: 'hello' is assigned a value but never used.
+If you want to contribute to this document please see the [Updating this
+manual](#updating-this-manual) section below.
 
-```
+Most guidelines listed here are recommendations, and the authors acknowledge
+that there will sometimes be valid exceptions.
+
+## Node versions
+
+> Only use Long Term Support (<abbr title="Long term support">LTS</abbr>)
+versions of Node.js.
+
+These are even-numbered versions (for example, Node.js 6.x or 8.x). However, it is important
+to keep an eye on the [Node.js LTS Schedule] as to when versions move in and out
+of LTS.
+
+> Don't use the `--harmony` or in-progress feature flags
```

Comment:
Agree we should add something about flags. I think the line perhaps should be more of a catch-all, to cover anything like --experimental features e.t.c